sci-physics/spheno/spheno-3.3.8.ebuild

```diff
@@ -33,9 +31,11 @@ src_compile() {
 src_install() {
 	dobin bin/${MY_PN}
 	# convenience symlink since the package is lowercase but the default produced binary is uppercase
-	dosym ${EPREFIX}/usr/bin/${MY_PN} /usr/bin/${PN}
+	dosym ${MY_PN} /usr/bin/${PN}
 	dolib.a lib/lib${MY_PN}.a
 	doheader include/*
-
-	use doc && dodoc doc/*
+	dodoc doc/*
+	docinto examples
+	dodoc input/*
+	docompress -x /usr/share/doc/${PF}/examples
```

Comment:
I added uncompressed examples to the doc folder.

Reply:
Great :+1: 

One last thing, could you please add a sign-off line to your commit in accordance with [Gentoo's copyright policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin)?

Reply:
I added it to my 3rd commit where I updated the metadata maintainers.